src/main/java/db/FileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package db;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    private static final String REPORT_PATH = "src/main/resources/dailyReport.csv";
```

Comment:
Are we going to create a new impl class each time when we need to write something to any another file?)